mtgjson5/set_builder.py

```diff
@@ -412,6 +413,12 @@ def build_mtgjson_set(set_code: str) -> Optional[MtgjsonSetObject]:
     mtgjson_set.tcgplayer_group_id = set_data.get("tcgplayer_id")
     mtgjson_set.booster = GitHubBoostersProvider().get_set_booster_data(set_code)
 
+    mtgjson_set.sealed_product = (
+        TCGPlayerProvider().generate_mtgjson_sealed_product_objects(
+            mtgjson_set.tcgplayer_group_id
```

Comment:
this will ping tcg api for each set, is that something ok to do? or should it be downloading the entirety of sealed stuff separately and then group them later?

Reply:
also some sets do not have tcgplayer_group_id, you can skip those i believe

Reply:
If sealed stuff can be downloaded at once, that might be better, re; Koda's info

Reply:
It is much harder and doesn't map well to how sets are currently built, also doesn't save many requests,